Jenkinsfile_CNP

```diff
@@ -13,29 +13,34 @@ def component = "processor"
 
 def channel = '#bsp-build-notices'
 
-List<LinkedHashMap<String, Object>> apiTestSecrets = [
-  // Base64-encoded PKCS 12 key store, containing client's private key and certificate known to the API
-  secret('test-valid-key-store', 'TEST_VALID_KEY_STORE'),
-  secret('test-valid-key-store-password', 'TEST_VALID_KEY_STORE_PASSWORD'),
+def apiTestSecrets = [
+  'bulk-scan-${env}': [
+    // Base64-encoded PKCS 12 key store, containing client's private key and certificate known to the API
+    secret('test-valid-key-store', 'TEST_VALID_KEY_STORE'),
+    secret('test-valid-key-store-password', 'TEST_VALID_KEY_STORE_PASSWORD'),
 
-  // Base64-encoded PKCS 12 key store containing private key and certificate known to the API, but expired
-  secret('test-expired-key-store', 'TEST_EXPIRED_KEY_STORE'),
-  secret('test-expired-key-store-password', 'TEST_EXPIRED_KEY_STORE_PASSWORD'),
+    // Base64-encoded PKCS 12 key store containing private key and certificate known to the API, but expired
+    secret('test-expired-key-store', 'TEST_EXPIRED_KEY_STORE'),
+    secret('test-expired-key-store-password', 'TEST_EXPIRED_KEY_STORE_PASSWORD'),
 
-  // Base64-encoded PKCS12 key store containing private key and certificate known to the API, but not yet valid
-  secret('test-not-yet-valid-key-store', 'TEST_NOT_YET_VALID_KEY_STORE'),
-  secret('test-not-yet-valid-key-store-password', 'TEST_NOT_YET_VALID_KEY_STORE_PASSWORD'),
+    // Base64-encoded PKCS12 key store containing private key and certificate known to the API, but not yet valid
+    secret('test-not-yet-valid-key-store', 'TEST_NOT_YET_VALID_KEY_STORE'),
+    secret('test-not-yet-valid-key-store-password', 'TEST_NOT_YET_VALID_KEY_STORE_PASSWORD'),
 
-  secret('test-subscription-key', 'TEST_SUBSCRIPTION_KEY'),
+    secret('test-subscription-key', 'TEST_SUBSCRIPTION_KEY'),
 
-  secret('test-storage-account-key', 'TEST_STORAGE_ACCOUNT_KEY')
+    secret('storage-account-primary-key', 'TEST_STORAGE_ACCOUNT_KEY')
```

Comment:
did the name of a secret in vault change?

Reply:
No it did not. Basically with this change/deprecation the presence of `test-config.tf` became obsolete - there is no need to add extra resources to vault with some `test-` prefix and just use them directly from vault specified in `secrets` definition in jenkins files ;)

In short: `test-storage-account-key` was just masking over the `storage-account-primary-key` from/in `bulk-scan` vault